egs/librispeech/ASR/pruned_transducer_stateless2/beam_search.py

```diff
@@ -36,7 +42,8 @@ def fast_beam_search_one_best(
     max_states: int,
     max_contexts: int,
     temperature: float = 1.0,
-) -> List[List[int]]:
+    return_timestamps: bool = False,
+) -> Union[List[List[int]], DecodingResults]:
```

Comment:
Please also update the docstring of the return value.

Reply:
Thanks. I will update that.